README.md.tpl

```diff
@@ -37,3 +36,4 @@ Configure your {brandname} cluster by specifying values in the `deploy.*` sectio
 | `deploy.resourceLabels` | Adds labels to {brandname} resources such as pods and services. | `{}` | - |
 | `deploy.makeDataDirWritable` | Allows write access to the `data` directory for each {brandname} Server node. | false | Setting the value to `true` creates an initContainer that runs `chmod -R` on the `/opt/infinispan/server/data` directory and changes its permissions. |
 | `deploy.nameOverride` | Specifies a name for all {brandname} cluster resources. | Helm Chart release name | Configure a name for the created resources only if you need it to be different to the Helm Chart release name. |
+| `deploy.infinispan` | The {brandname} server configuration. | - | Care should be taken when modifying the default yaml to ensure that the Chart continues to function as expected. It is not recommended to modify the "metrics" property realm, endpoint or socket binding configuration. |
```

Comment:
```suggestion
| `deploy.infinispan` | {brandname} Server configuration. | - | Modifies the underlying server instance of each pod. You should not change the default endpoints, socket bindings, or the security realm named "metrics". Modifying these default properties can result in unexpected behavior and loss of service.|
```

Reply:
@ryanemerson does "unexpected behaviour and loss of service" sound like too much?